xml/planning-architecture-input_model-other_topics-services_components.xml

```diff
@@ -129,6 +135,13 @@ swift-ring-builder
 swift-rsync</screen>
      </entry>
     </row>
+    <row>
+     <entry>HPE Virtual Storage Array</entry>
+<!--     <entry></entry>-->
+     <entry>vsa-storage</entry>
+     <entry><screen>cmc-service
+vsa</screen></entry>
+    </row>
```

Comment:
*HPE Virtual Storage Array* entry actually shouldn't be in the table. Could you please remove it? Thanks! :smile: